crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -182,6 +186,175 @@ pub enum WalletInfo {
     GooglePay(GpayInfo),
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+#[serde(untagged)]
+pub enum BankRedirectInfo {
+    Ideal(IdealInfo),
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+pub struct IdealInfo {
+    pub issuer_id: Option<String>,
+}
+
+pub struct MultisafepayBankNames<'a>(&'a str);
```

Comment:
Can you convert this to an enum. You can use [serde rename](https://serde.rs/field-attrs.html) here.

Reply:
Can you suggest me a way to handle banks which are not supported ?

Reply:
Completed the change @srujanchikke